picard/tagger.py

```diff
@@ -118,7 +118,14 @@ def __init__(self, picard_args, unparsed_args, localedir, autoupdate):
         # Allow High DPI Support
         self.setAttribute(QtCore.Qt.AA_UseHighDpiPixmaps)
         self.setAttribute(QtCore.Qt.AA_EnableHighDpiScaling)
-
+        super().setStyleSheet(
```

Comment:
Can't really test this as it doesn't affect me. Maybe @vishichoudhary can test and see if it fixes it for him too?

Reply:
@zas @sambhav no effect of adding these two
```
 subcontrol-origin: margin;
 subcontrol-position: top left;
```
this too works for me 

```
 super().setStyleSheet(
             """ 
             QGroupBox::title { }
             """
) 
```

weird.

Reply:
@vishichoudhary : i changed the patch to use just this, it works for me too.

Reply:
Isn't this a bit too weird and unpredictable?

Reply:
Yes it is... but it works ... don't ask me why.